index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Elson Otake</title>
+</head>
+<body>
+  <header class="header">
+    <ul class="header_list">
+      <li class="company">Origami Inc</li>
+      <li class="menu">
+          <span class="material-symbols-rounded" id="hamburger">menu</span>
+      </li>
+    </ul>
+  </header>
+  <section class="presentation">
+    <div class="pres">
+      <h2 class="name">Elson Otake</h2>
+      <h4 class="position">Software Developer</h4>
+      <p class="descr">I can help you build a product , feature or website<br>Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    </div>
+  </section>
+  <hr class="line_1">
+  <section class="qualif_group">
+    <div class="qualif">
+      <h6 class="qualif_title">Language</h6>
+      <ul>
+        <li class="qualif_item">Javascript</li>
+        <li class="qualif_item">Ruby</li>
+        <li class="qualif_item">Html</li>
+        <li class="qualif_item">Css</li>
+      </ul>
+    </div>
+    <div class="qualif">
+      <h6 class="qualif_title">Frameworks</h6>
+      <ul>
+        <li class="qualif_item">Bootstrap</li>
+        <li class="qualif_item">Ruby on Rails</li>
+        <li class="qualif_item">RSPec</li>
+        <li class="qualif_item">CapyBara</li>
+        <li class="qualif_item">Selenium</li>
+      </ul>
+    </div>
+    <div class="qualif">
+      <h6 class="qualif_title skills">Skills</h6>
+      <ul>
+        <li class="qualif_item">Codekit</li>
+        <li class="qualif_item">Github</li>
+        <li class="qualif_item">Codepen</li>
+        <li class="qualif_item">Gitlab</li>
+        <li class="qualif_item">Terminal</li>
+      </ul>
+    </div>
+  </section>
+  <hr class="line_2">
+  <div class="links_group">
+    <div class="links">
+      <img class="icon" src="images/icons8-github-30.png" alt="github icon from icons8"/>
+      <img class="icon" src="images/icons8-linkedin-30.png" alt="linkedin icon from icons8"/>
+      <img class="icon" src="images/icons8-mão-fazendo-sinal-de-paz-30.png" alt="peace icon from icons8"/>
+      <img class="icon" src="images/icons8-twitter-30.png" alt="twitter icon from icons8"/>
+      <img class="icon" src="images/icons8-circled-m-30.png" alt="M icon from icons8"/>
+    </div>
```

Comment:
- [ ] You can use a `ul` tag here to wrap the social media links that have multiple links for each social media link. You can use something like the following:
```
<ul>
   <li>
      <a>
         <img> first image
      </a>
   </li>
   <li>
      <a>
         <img> second image
      </a>
   </li>
   ...
</ul>
```